solutions/usaco-993.mdx

```diff
@@ -0,0 +1,82 @@
+---
+id: usaco-993
+source: USACO Gold 2020 January
+title: Time is Mooney
+author: Nathan Gong
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_time_gold_jan20.html)
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(T_{\text{max}} \cdot (N + M))$
+
+<LanguageSection>
+
+<JavaSection>
+
+```java
+import java.io.*;
+import java.util.*;
+
+public class TimeIsMooney {
+	public static void main(String[] args) throws IOException {
+		Scanner sc = new Scanner(new File("time.in"));
+		PrintWriter out = new PrintWriter("time.out");
+
+		// max time that we check (it can be any arbitrary large number)
```

Comment:
...but if we made it any magnitude larger it would TLE

Reply:
true, i just wanted to specify that the number is arbitrary. is there a better way to word it?

Reply:
just say something like "it wouldn't make sense to check any larger number"
and add a quick multiline comment (2-3 lines) on why it'd be stupid to do so

Reply:
is what i added sufficient

Reply:
@SansPapyrus683 ^

Reply:
oh sorry lol lemme check